tests/test_user_agent.py

```
@@ -0,0 +1,33 @@
+        # Verify that Client was called with the correct parameters
+        mock_client.assert_called_once_with(
+            "http://localhost:7700",
+            "test_key",
+            client_agents=("meilisearch-mcp", f"v{__version__}"),
+        )
+
+
+def test_user_agent_includes_correct_version():
+    """Test that the user agent includes the correct version from __version__.py"""
+    with patch("src.meilisearch_mcp.client.Client") as mock_client:
+        client = MeilisearchClient()
```

Ruff (0.11.9)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)